### DISCUSSION TITLE
自主决定在哪个显卡上运行

### DISCUSSION THREAD

**Discussion by TTvTT** [ywmoyue/biliuwp-lite#653] [feature]
### 问题描述

自主决定在哪个显卡上运行，貌似windows自带的显示设置并不管用

### 描述解决方案

貌似windows自带的显示设置并不管用

### 代码贡献

否

**Comment by GD-Slime** [comment]
使用的设备是什么呢？是否是带核显和独显的笔记本？如果是的话可以试试关掉独显直连

**Comment by HeMuxD** [comment]
从Windows设置里面设置一下就行，所有的程序包括UWP应用都可以设置
